enki/PhysicalEngine.h

```diff
@@ -448,6 +460,8 @@ namespace Enki
 		
 		//! All the objects in the world
 		Objects objects;
+		//! A list to defined object for which the physics will be skipped
+		Objects SkipPhysicsObjectsContainer;
```

Comment:
Yes this is possible
I was thinking to make this "attribute" (invisible in physics engine), more standart but it's true that you don't need it in your application.